client/app/dedicatedCloud/datacenter/drp/steps/common/final/dedicatedCloud-datacenter-drp-finalStep.controller.js

```diff
@@ -0,0 +1,111 @@
+import template from './dedicatedCloud-datacenter-drp-finalStep.html';
+
+export default class {
+  /* @ngInject */
+  constructor(
+    $q, $state, $stateParams, $translate, $uibModal,
+    Alerter, DedicatedCloudDrp, OvhApiMe,
+    DEDICATEDCLOUD_DATACENTER_DRP_STATUS,
+  ) {
+    this.$q = $q;
+    this.$state = $state;
+    this.$stateParams = $stateParams;
+    this.$translate = $translate;
+    this.$uibModal = $uibModal;
+    this.Alerter = Alerter;
+    this.DedicatedCloudDrp = DedicatedCloudDrp;
+    this.OvhApiMe = OvhApiMe;
+    this.DEDICATEDCLOUD_DATACENTER_DRP_STATUS = DEDICATEDCLOUD_DATACENTER_DRP_STATUS;
+  }
+
+  $onInit() {
+    this.loading = true;
+    this.drpInformations = this.$stateParams.drpInformations;
+    const { state } = this.drpInformations;
+    const otherPccInformations = this.getOtherPccInformations();
+
+    return this.$q.all({
+      enableDrp: this.DEDICATEDCLOUD_DATACENTER_DRP_STATUS.deliveredOrProvisionning.includes(state)
+        ? this.$q.when({})
+        : this.DedicatedCloudDrp.enableDrp(this.drpInformations),
+      me: this.OvhApiMe.v6().get().$promise,
+      secondaryPccState: _.isNull(otherPccInformations.serviceName)
+        ? this.$q.when({})
+        : this.DedicatedCloudDrp.getDrpState(otherPccInformations),
+    })
+      .then(({ enableDrp, me, secondaryPccState }) => {
+        if (enableDrp.state === 'todo') {
+          this.isEnabling = true;
+        }
+        this.email = me.email;
+
+        this.deleteActionAvailable = secondaryPccState
+          .state === this.DEDICATEDCLOUD_DATACENTER_DRP_STATUS.delivered;
+        if (state === this.DEDICATEDCLOUD_DATACENTER_DRP_STATUS.delivered) {
+          this.Alerter.success(`
+            <p>${this.$translate.instant('dedicatedCloud_datacenter_drp_confirm_create_success_part_one')}</p>
+            <strong>${this.$translate.instant('dedicatedCloud_datacenter_drp_confirm_create_success_part_two', { email: this.email })}</strong>
+          `, 'dedicatedCloudDatacenterDrpAlert');
+        }
+      })
+      .catch((error) => {
+        this.Alerter.error(`${this.$translate.instant('dedicatedCloud_datacenter_drp_confirm_create_error')} ${_.get(error, 'message', '')}`, 'dedicatedCloudDatacenterDrpAlert');
+      })
+      .finally(() => {
+        this.loading = false;
+      });
+  }
+
+  getOtherPccInformations() {
+    const primaryOptions = {
+      serviceName: _.get(this.drpInformations, 'primaryPcc.serviceName', null),
```

Comment:
Wouldn't `undefined` work ?
```suggestion
      serviceName: _.get(this.drpInformations, 'primaryPcc.serviceName'),
```